src/coreclr/inc/jiteeversionguid.h

```diff
@@ -43,11 +43,11 @@ typedef const GUID *LPCGUID;
 #define GUID_DEFINED
 #endif // !GUID_DEFINED
 
-constexpr GUID JITEEVersionIdentifier = { /* 3a8a07e7-928e-4281-ab68-cd4017c1141b */
-    0x3a8a07e7,
-    0x928e,
-    0x4281,
-    {0xab, 0x68, 0xcd, 0x40, 0x17, 0xc1, 0x14, 0x1b}
+constexpr GUID JITEEVersionIdentifier = { /* 95c688c7-28cf-4b1f-922a-11bf3947e56f */
```

Comment:
Side question: now that runtime and corert are in a monorepo, can we replace `JITEEVersionIdentifier` with computed hash without needing to hardcode in the sources?

```sh
$ git ls-files -s src/coreclr/vm/jitinterface.h | while read _ hash __; do
    printf 'a "more" universal JIT interface identifier: %s' "$hash"
done

a "more" universal JIT interface identifier: cc2cccca72a0961f75ecb50fa79b07888ebec164
```